client.py

```diff
@@ -897,7 +901,16 @@ def __call__(self, window, username, password, settings):
 
 class LoginHandler(Ui_LoginWindow):
     def __init__(self, window):
-        self.settings = config.Config("settings.json", default={"blocked_users": [], "server_ip": "", "server_port": "", "should_render_markdown": True})
+        self.settings = config.Config(
+            "settings.json",
+            default={
+                "blocked_users": [],
+                "server_ip": "",
+                "server_port": "",
+                "should_render_markdown": True,
+                "secure_connect": False
```

Comment:
Why is this `False`?

Reply:
I believe at the time of this commit, the server didn’t have `wss`. Citation needed